offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
+
+# Main HTML file
+index.html
+
+# Translations
+cz/index.html
+da/index.html
+de/index.html
+es/index.html
+fr/index.html
+ko/index.html
+pl/index.html
+pt/index.html
+ru/index.html
+zh/index.html
+
+# CSS Files
+css/critical.css
```

Comment:
I think we don’t need to cache the critical CSS file if we cache the full one. What do you think?